audit/doc.go

```diff
@@ -21,3 +21,19 @@ they make a list and get these AuditResultStructures from the audit contract.
 Audit contract does not produce notifications to process.
 */
 package audit
+
+/*
+Contract storage model.
+
+# Summary
+Key-value storage format:
+ - [64]byte -> []byte
+   Data audit results encoded into NeoFS API binary protocol format. Results are
+   identified by triplet concatenation:
+    1. 8-byte little-endian unsigned integer NeoFS epoch when audit was performed
```

Comment:
That's not correct, the length in fact is dynamic.

Reply:
dropped length